src/i18n/en.json

```diff
@@ -57,6 +57,10 @@
     "isPublic": "Data public?",
     "makePublic": "Make installation data public?",
     "otherInstallations": "Other installations of sensor",
+    "public": {
+      "true": "Public",
+      "false": "Private"
```

Comment:
In my opinion "Not public" would be clearer.
Otherwise as a user I might wonder what a "private installation" implies. (Especially in the context of an organisation "private" seems unfitting to me.)

It's a minor point, if you disagree, I'm happy to merge it anyways!

Reply:
I put the translation string in a hierarchy, so it can be more specific to its use. Here, the use is in tables, where the header states "Data public?" In this setting, simply filling the table column with "public" and "private" should be clear and concise.